spark/dl/src/test/scala/com/intel/analytics/bigdl/optim/DistriOptimizerSpec.scala

```diff
@@ -157,7 +200,7 @@ class DistriOptimizerSpec extends FlatSpec with Matchers with BeforeAndAfter {
     mm.getParameters()._1.fill(0.125)
     val optimizer = new DistriOptimizer[Double](mm, dataSet, new MSECriterion[Double]())
       .setState(T("learningRate" -> 20.0))
-      .setEndWhen(Trigger.maxEpoch(5))
```

Comment:
@yiheng I'm not sure there. @sethah changed `maxEpoch` from `5` to `1`. I think this `maxEpoch` should be 2 at least. But why you set a `5`?